### PR TITLE
feat: spade more food

### PR DESCRIPTION
**Pull request by midgleyc**

### CODE REVIEW

**Review by gausie — APPROVED** [review]

File: src/data/fullness.txt [hunk 1]
```diff
@@ -98,7 +98,7 @@ bowl of eyeballs	2	1	awesome	7-9	0	10-20	0	30 Your Eyes are Peeled! (+25% item d
 bowl of fishysoisse	6	8	awesome	27-29	48-74	67-70	52-70
 bowl of lucky charms	1	1	crappy	1	0	6-9	0	50% chance of leprechaun hatchling
 bowl of maggots	2	1	EPIC	10-12	10-20	0	0
-bowl of mernudo	3	1	good	0	0	0	0	Unspaded 30 Sea Guts
+bowl of mernudo	3	1	good	9	0	0	0
```
Comment (Veracity0) [inline src/data/fullness.txt:101]: Isn't it still 30 Sea Guts?

File: src/data/fullness.txt [hunk 2]
```diff
@@ -622,7 +622,7 @@ nut-shaped Crimbo cookie	2	1	awesome	7-11	0	0	0	20 Nut-Rition (+100% all stats)
 nutty organic salad	3	11	awesome	12-15	0	31-40	0	SALAD
 occult jelly donut	1	4	awesome	3-5	0	40-60	0	30 Filled With Magic (+100% spell dmg)
 oily golden mushroom	0	1	crappy	0	0	0	0	Inedible
-ojo de pez burrito	3	1	good	0	0	0	0	Unspaded 30 Fish-Eye View
+ojo de pez burrito	3	1	good	9	0	0	0
```
Comment (Veracity0) [inline src/data/fullness.txt:625]: 30 Fish-Eye View

**Review by Veracity0 — CHANGES_REQUESTED** [review]

File: src/data/fullness.txt (hunk 1 above)
Comment (midgleyc) [reply src/data/fullness.txt:101]: no, this went away after Crimbo 2019 ended

File: src/data/fullness.txt (hunk 2 above)
Comment (midgleyc) [reply src/data/fullness.txt:625]: no, this went away after Crimbo 2019 ended

**Review by Veracity0 — APPROVED** [review]

**Pull request merged**